.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug Report
+about: Create a report to help us solve bugs!
+labels: bug
+---
```

Comment:
```suggestion
---
name: Feature Request
---
```

Reply:
> Otherwise, @Freyskeyd proposed to use Github Discussions instead for feature request, but switching to the relevant Linear item for further discussion on that topic

I fixed the copy/paste mistake. As to the wider discussion, we can wait to conclude it on Linear.